packages/ra-input-rich-text/src/RichTextInput.stories.tsx

```diff
@@ -187,6 +191,61 @@ export const Validation = (props: Partial<SimpleFormProps>) => (
     </AdminContext>
 );
 
+const MyRichTextInputToolbar = ({ ...props }) => {
+    return (
+        <RichTextInputToolbar {...props}>
+            <FormatButtons />
+        </RichTextInputToolbar>
+    );
+};
+
+export const EditedToolbar = (props: Partial<SimpleFormProps>) => (
```

Comment:
Please rename that story to the name of the prop you're testing (in this case, `Toolbar`)

Reply:
done